src/parser/monk/mistweaver/modules/spells/InvokeChiJi.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class InvokeChiJi extends Analyzer {
+  static dependencies = {
+    conbatants: Combatants
+  };
+
+  petID = null;
+  healing = 0;
+  overhealing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.GUST_OF_MISTS_CHIJI), this.handleGust);
+    debug && this.addEventListener(Events.fightend, this.fightEndDebug);
+  }
+
+  handleGust(event) {
+    this.healing += (event.amount || 0) + (event.absorbed || 0);
+  }
+
+  fightEndDebug() {
```

Comment:
This debug currently does nothing; just at the end of the fight says chi-ji summoned (regardless if it was or not). Id recommend removing it and the eventlistener on L28